maps/RoidStation.dmm

```diff
@@ -8020,12 +8067,15 @@
 "cZh" = (/obj/machinery/computer/general_air_control/large_tank_control{frequency = 1441; input_tag = "n2_in"; name = "Nitrogen Supply Control"; output_tag = "n2_out"; sensors = list("n2_sensor" = "Tank")},/obj/machinery/atmospherics/pipe/simple/filtering/visible{dir = 4},/turf/simulated/floor{icon_state = "red"},/area/engineering/atmos)
 "cZi" = (/obj/machinery/light,/obj/machinery/atmospherics/trinary/filter/mirrored{dir = 4; filter_type = 2; icon_state = "hintactm_on"; name = "Gas filter (Nitrogen tank)"; on = 1},/turf/simulated/floor{icon_state = "red"; dir = 6},/area/engineering/atmos)
+"cZj" = (/obj/machinery/atmospherics/pipe/manifold/supply/hidden{dir = 4; initialize_directions = 13},/obj/structure/bed/chair/comfy/black{dir = 1},/turf/simulated/floor{icon_state = "dark vault full"},/area/security/lobby)
 "cZk" = (/obj/machinery/light,/obj/machinery/atmospherics/trinary/filter/mirrored{dir = 4; filter_type = 3; icon_state = "hintactm_on"; name = "Gas filter (CO2 tank)"; on = 1},/turf/simulated/floor{dir = 1; icon_state = "dark floor corner"},/area/engineering/atmos)
 "cZl" = (/obj/machinery/atmospherics/pipe/simple/filtering/visible{dir = 4},/obj/machinery/atmospherics/pipe/simple/yellow/visible,/turf/simulated/floor{dir = 10; icon_state = "purple"},/area/engineering/atmos)
+"cZm" = (/obj/machinery/newscaster/security_unit{pixel_x = 27; pixel_y = 2},/obj/machinery/computer/security,/turf/simulated/floor{icon_state = "dark vault stripe"},/area/security/lobby)
 "cZn" = (/obj/machinery/light,/obj/machinery/atmospherics/trinary/filter/mirrored{dir = 4; filter_type = 0; icon_state = "hintactm_on"; name = "Gas filter (Plasma tank)"; on = 1},/turf/simulated/floor{tag = "icon-purple (SOUTHEAST)"; icon_state = "purple"; dir = 6},/area/engineering/atmos)
 "cZo" = (/obj/machinery/atmospherics/pipe/simple/filtering/visible{dir = 4},/obj/machinery/atmospherics/pipe/simple/yellow/visible,/turf/simulated/floor{tag = "icon-bluered (SOUTHWEST)"; icon_state = "bluered"; dir = 10},/area/engineering/atmos)
 "cZp" = (/obj/machinery/computer/general_air_control/large_tank_control{frequency = 1441; input_tag = "n2o_in"; name = "Nitrous Oxide Supply Control"; output_tag = "n2o_out"; sensors = list("n2o_sensor" = "Tank")},/obj/machinery/atmospherics/pipe/simple/filtering/visible{dir = 4},/turf/simulated/floor{tag = "icon-bluered"; icon_state = "bluered"},/area/engineering/atmos)
 "cZq" = (/obj/machinery/atmospherics/trinary/filter/mirrored{dir = 4; filter_type = 4; icon_state = "hintactm_on"; name = "Gas filter (N2O tank)"; on = 1},/turf/simulated/floor{tag = "icon-bluered (SOUTHEAST)"; icon_state = "bluered"; dir = 6},/area/engineering/atmos)
+"cZr" = (/turf/simulated/floor,/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 9},/turf/simulated/floor{tag = "icon-siding2"; icon_state = "siding2"},/area/crew_quarters/fitness)

```

Comment:
Why are there two floors? Why does one have a tag and what is `"icon-siding2"`?

Reply:
Because those sidings are coded oddly. They overlap but otherwise it has no effect on game play because they work like regular floors. had them like that since the station launched and it hasn't broken anything.